draft-irtf-cfrg-vdaf.md

```diff
@@ -4609,6 +4615,37 @@ differential privacy.
 > TODO(issue #94) Describe (or point to some description of) the central DP
 > mechanism for Poplar described in {{BBCGGI21}}.
 
+### Unsafe IDPF-Tree Traversals
+
+The secure sketch described in {{poplar1}} is used by the Aggregators to check
+that the shares of the vector obtained by evaluating a Client's IDPF at a
+sequence of candidate prefixes has at most one non-zero entry, and that the
+non-zero entry is `1`. Indeed, for an honestly generated Poplar1 report, we
+expect the IDPF tree has exactly one non-zero path from the root to the leaves,
+but this is not guaranteed by the secure sketch alone.
+
+IDPFs are designed to prevent a malicious Client from producing key shares with
+more than one non-zero path in the tree. This property is formalized as
+"extractability" in Appendix D of {{BBCGGI21}}: for any pair of key shares and
+public share produced by a malicious Client and a given level of the tree, one
+can efficiently extract from the Client the underlying prefix (i.e., a prefix
+of `alpha`), in particular from its queries to the random oracle.
```

Comment:
@schoppmp a question: I wonder if we should be worried about using `XofFixedKeyAes128` for generating leaf nodes, since the Poplar paper wants this to be a random oracle?